### DISCUSSION TITLE
Turtle Elvator

### DISCUSSION THREAD

**Discussion by SteveMan67** [cc-tweaked/CC-Tweaked#1882] [Q&A]
I know that you can stand on a turtle and you can move on it. would it then be possible to use it as an elevator to go into the mines?

**Comment by SquidDev** [comment]
Yep! It will be quite slow (turtles are not speedy beasts), but definitely possible!

**Comment by fatboychummy** [comment]
Yes, but be careful: Server or client lag can cause you to fall through the turtle while it's moving.

**Comment by zyxkad** [comment]
Turtle works like piston, but it has smaller hitbox. So you will have a chance cross the turtle and fall out when you are being pushed up

**Comment by SteveMan67** [comment]
I was thinking of a 1x1 shaft for the turtle. I'm not sure if that's big enough of a space to fall through. I was picturing a pocket computer that controlled it remotely (so you could call it to your level) as well. I would definitely place a water source at the bottom in case of a fall though.
